.pipelines/stages/jobs/steps/capi-linux-step.yml

```diff
@@ -66,12 +80,22 @@ steps:
   - template: utils/capi-archive.yml
     parameters:
       archiveType: tar
+  - script: |
+      set -e -x
+      docker run \
+      --rm \
+      --volume $(Build.Repository.LocalPath):/ort_genai_src \
+      -w /ort_genai_src/ onnxruntime$(ep)build$(arch) \
```

Comment:
Will this docker name conflict with the ones in the  main ort repo?

Reply:
update the name